src/network/peers.rs

```diff
@@ -1181,6 +1188,19 @@ impl<N: Network, E: Environment> Peer<N, E> {
                                         break;
                                     }
 
+                                    // Procure a fast, short hash of the block hash, and check if it's present in the cache.
+                                    let fast_hash = {
+                                        let mut hasher = RandomXxHashBuilder64::default().build_hasher();
```

Comment:
Is there anything from `std` lib that can be used with similar behavior?

Reply:
not from the `std`; there's alternatives, but usually not that fast or high-quality; I'm open to using an alternative way of uniquely identifying blocks, though